docs/get-started/Velox.md

```diff
@@ -314,8 +314,7 @@ First of all, compile gluten-delta module by a `delta` profile, as follows:
 mvn clean package -Pbackends-velox -Pspark-3.3 -Pdelta -DskipTests
 ```
 
-Then, put the additional `gluten-delta-XX-SNAPSHOT.jar` to the class path (usually it's `$SPARK_HOME/jars`).
-The gluten-delta jar is in `gluten-delta/target` directory.
+Once built successfully, delta features will be included in gluten-velox-bundle-X jar.
```

Comment:
Put line `318` together with this line please. Can you also help update the content for iceberg in this doc? seems there is some duplicated contents.
```
Once built successfully, iceberg features will be included in gluten-velox-bundle-X jar. Then you can query iceberg table by gluten/velox without scan's fallback.

After the two steps, you can query iceberg table by gluten/velox without scan's fallback.
```

Reply:
Sure, it's done.